carma_planning_msgs/srv/GetEmergencyRoute.srv

```diff
@@ -0,0 +1,20 @@
+#
+# GetEmergencyRoute.srv
```

Comment:
I believe a GetEmergencyRoute.srv will need to be created within cav_srvs/srv as well in order to ensure this service is bridged between ROS1 and ROS2.